src/raster/rasterband.rs

```diff
@@ -467,8 +467,8 @@ impl<'a> RasterBand<'a> {
                 window_size.0 as c_int,
                 window_size.1 as c_int,
                 data.as_mut_ptr() as *mut c_void,
-                size.0 as c_int,
-                size.1 as c_int,
+                shape.0 as c_int,
```

Comment:
Let's use `try_into` or `try_from` here (unless you've already changed it in another commit). Same for `window` and `window_size`.

Reply:
Maybe we should have made them `u16`!??‽? 😮

Reply:
Sorry, what do you mean?